### ISSUE TITLE
Error running on JDK 11/12 as missing JavaFX

### ISSUE THREAD

**Issue by AVTConnect** [yapplications/ADB-GUI#6]
When running the .jar on JDK 11/12 I get the following error: 

```
Error: Could not find or load main class application.Main
Caused by: java.lang.NoClassDefFoundError: javafx/application/Application
```
It seems that JavaFX has not been included in JDK 11/12. See https://www.infoworld.com/article/3305073/removed-from-jdk-11-javafx-11-arrives-as-a-standalone-module.html

**Comment by tripkin** [comment]
This is still a problem. Is there a workaround that can be used without installing an older version of java?

**Comment by JoeBeeton** [comment]
Yes its not working for me either. I believe it would be possible to create a fat jar including the required javafx stuff relatively easily. Unfortunately there is a open PR from 2 years ago so I don't think any new PRs be approved and there is no license attached to this project so I cant fork it :(

**Comment by lisitso** [comment]
How to run the jar? The error still comes up

**Comment by AndKe** [comment]
Is there a workaround for use with current Java releases?

**Comment by diVineProportion** [comment]
as @AVTConnect mentioned, JavaFX doesn't appear to be shipped with versions after 7/8. https://www.oracle.com/java/technologies/javase/javafx-overview.html gives you directions on how to implement JavaFX in 11/12 but it's far easier to just download jdk-8u291-windows-x64.exe

dl, install, run
